spring-cloud-task-samples/helloworld/src/main/java/demo/SampleTaskConfiguration.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package demo;
+
+import org.springframework.cloud.task.annotation.EnableTask;
+import org.springframework.context.annotation.Configuration;
+
+
+/**
+ * Enables Spring Cloud Task.
+ *
+ * @author Glenn Renfro.
+ */
+@Configuration
+@EnableTask
+public class SampleTaskConfiguration {
```

Comment:
Is this class needed?  Can't we just drop `@EnableTask` onto the main class (and if not, that's probably a bug)?